packages/amplify_storage_s3/ios/Classes/FlutterGetUrlRequest.swift

```diff
@@ -24,9 +24,11 @@ struct FlutterGetUrlRequest {
         self.options = setOptions(request: request)
     }
     
-    static func isValid(request: Dictionary<String, AnyObject>) -> Bool {
-        let valid: Bool = request["key"] is String
-        return valid
+    static func isValid(request: Dictionary<String, AnyObject>) throws {
+        let validationErrorMessage = "GetURL request malformed."
+        if !(request["key"] is String) {
+            throw AmplifyFlutterValidationException(errorDescription: validationErrorMessage, recoverySuggestion: "key is missing.")
```

Comment:
Same here, recoverySuggestion of "key is missing" seems incomplete. Is this useful for customers to fix this error?

Reply:
Changing to "Ensure that {key/path} variable is present in request."